breathecode/admissions/views.py

```diff
@@ -1177,6 +1177,9 @@ def post(self, request, academy_id=None):
         for key in request.data:
             data[key] = request.data.get(key)
 
+        if 'is_hidden_on_prework' not in data or data['is_hidden_on_prework'] == 'academy_default':
```

Comment:
You don't need copy the value from the academy, if it is none it will to inherit the value of it, you think the default value should changed and we don't know if the cohort is marked with this value from inheritance or for the will of the user